test/src/xquery/xquery3/fnSort.xql

```diff
@@ -0,0 +1,172 @@
+xquery version "3.1";
+
+module namespace testSort="http://exist-db.org/xquery/test/function_sort";
+
+declare namespace test="http://exist-db.org/xquery/xqsuite";
+
+declare
+    %test:assertEquals("1","3","4","5","6")
+function testSort:single_parameter() {
+    fn:sort((1, 4, 6, 5, 3))
```

Comment:
I see that the test suite itself does some type conversion: "If the function returns an atomic type, the assertion argument is cast into the same type and the two are compared using the eq operator" (http://exist-db.org/exist/apps/doc/xqsuite.xml#D1.2.6). Unless we're testing the test suite's own ability to do this type conversion, I'm not sure this test is needed. The tests that compare like types (integer-to-integer, string-to-string) should suffice.